docs/how-to-guides/use-a-custom-timestamp-column.md

```diff
@@ -0,0 +1,213 @@
+# How to use a custom timestamp column
+
+## Context
+
+You have events that have a timestamp attribute. You want to do time-related processing over them and want ksqlDB to use those timestamps for processing. Because ksqlDB defaults to using wall clock time, you need to tell ksqlDB where to find the timestamps within the events. This is known as "processing-time" (using the wall clock) and "event-time" (using timestamps within events).
```

Comment:
```suggestion
You have events that have a timestamp attribute. You want to do time-related processing over them and want ksqlDB to use those timestamps for processing. Because ksqlDB defaults to using wall clock time, you need to tell ksqlDB where to find the timestamps within the events. This is known as "processing-time" (using the wall clock) and "event-time" (using timestamps within events).
```
```suggestion
You have events that have a timestamp attribute. You want to do time-related processing over them and want ksqlDB to use those timestamps for processing. Because ksqlDB defaults to using wall clock time, you need to tell ksqlDB where to find the timestamps within the events. This is known as *processing-time* (using the wall clock) and *event-time* (using timestamps within events).
```